### DISCUSSION TITLE
How to use cmake functions with pybind11 is installed via conan-center

### DISCUSSION THREAD

**Discussion by davidhjp01** [pybind/pybind11#3485] [Q&A]
I do not see config or module cmake files installed when I download pybind11 package via conan-center so when I have `find_package(pybind11 REQUIRED)`, cmake cannot find pybind11. Can I use cmake functions when pybind11 is installed via conan-center?

**Comment by CPickens42** [comment]
Hello, I ended up using this ugly patch: https://github.com/conan-io/conan-center-index/issues/9343#issuecomment-1068921461
I would love also to have a better way of handling that.
